class.jetpack.php

```diff
@@ -7249,6 +7249,15 @@ public static function show_backups_ui() {
 		return self::is_plugin_active( 'vaultpress/vaultpress.php' ) || apply_filters( 'jetpack_show_backups', true );
 	}
 
+	/**
+	 * Returns a boolean for whether the Setup Wizard should be displayed or not.
+	 *
+	 * @return bool True if the Setup Wizard should be displayed, false otherwise.
+	 */
+	public static function show_setup_wizard() {
+		return apply_filters( 'jetpack_show_setup_wizard', false ) && self::is_active();
```

Comment:
You'll want to add a docblock to that filter to keep the parser happy and allow it to create a codex page for that new filter.
You can check https://github.com/Automattic/jetpack/blob/add/setup-wizard-intro-page/_inc/lib/admin-pages/class.jetpack-react-page.php#L141-L147 for an example.

Reply:
Thanks, done!